src/gengowatcher/web.py

```
@@ -483,6 +753,397 @@ async def accept_job(self, job_id: str) -> bool:
+            entry = self.save_uploaded_file(
+                self._filename_from_download_response(url, response),
+                response.content,
+                content_type=response.headers.get("content-type"),
+                job_id=job_id,
+            )
+            self._handle_stored_file_for_job(
+                entry,
+                content=response.content,
+                mode="auto",
+            )
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)